src/mime/media_type.cr

```diff
@@ -447,8 +447,8 @@ module MIME
     end
 
     private def self.decode_rfc2231(encoded : String)
-      encoding, split1, rest = encoded.partition('\'')
-      lang, split2, value = rest.partition('\'')
+      encoding, _split_one, rest = encoded.partition('\'')
+      _lang, _split_two, value = rest.partition('\'')
```

Comment:
I think we can just use underscore without names to make the relevant variable names pop out more.
```suggestion
      encoding, _, rest = encoded.partition('\'')
      _, _, value = rest.partition('\'')
```
Maybe keep `_lang` as a label, but I'd certainly toss out the separator names.

Reply:
`_lang` would still be unused. Does Ameba intentionally skip over these variables? (I know that Elixir does, for example.)

Reply:
Yes, I think the purpose of variable names with underscore prefix is to indicate that variable is unused, but still give it a label to document context.

Reply:
I fixed `split` word, and use the lang with the `_` prefix, linter doesn't worry about this line now